src/microbe_stage/Microbe.cs

```diff
@@ -420,14 +425,17 @@ public override void _Process(float delta)
                 }
 
                 totalMovement += queuedMovementForce;
-
```

Comment:
Unnecessary blank line removal.

Reply:
sorry whats the problem there? isnt it the same logic?

Reply:
Blank lines are friends, not enemies. They should not be erased without a reason.

Reply:
i dont understand the logic for this blank line.

Reply:
There's no reason really to remove it, so it'd be better to just let it be.
It still makes logical sense to me to keep the parts building totalMovement separate from when then the movement force is applied. For example if someone wants to add a new force, they can see the blank line after line 427 and add it there.